booking-engine-widget/use-cases/disable-widget-caching.md

```diff
@@ -0,0 +1,61 @@
+# Do not use a booking engine script cached by your server
```

Comment:
I believe the page should be titled **Disabling widget caching**, I understand the warning is important so I would put that as a breakout at the top, like this ->
```suggestion
# Disabling widget caching

> **Important:** Do not use a booking engine script cached by your server!
```